canonical-web-and-design/maas-ui
# refactor(ui): update machine header to support tag forms

## Done

- Separate the main machine/pool/tag view and header so that it can support tag header forms.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the machine view and check that you can open header forms.
- Click on the pools tab and check that clicking on the add pool button opens the form.
- Click on the tags tab and check that clicking on the add tag button opens the placeholder form in the header.

## Fixes

Fixes: canonical-web-and-design/app-tribe#741.
